pitop/processing/algorithms/__init__.py

```diff
@@ -1 +1,2 @@
 from .line_detect import process_frame_for_line
+from .ball_detect import BallDetector
```

Comment:
Should we make it clear that these are 'image processing' algorithms?

Reply:
yeah we probably should, do you mean a separate directory or just adding comments in this __init__?